packages/grafana-data/src/themes/context.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import { createTheme } from './createTheme';
+
+// Use Grafana Dark theme by default
```

Comment:
**Nit:** I would probably remove this comment from here. The reason is that it's kinda disconnected from the place [where the dark default is set](https://github.com/grafana/grafana/blob/6cede325354d67b940d582be1e4f25eb1df26694/packages/grafana-data/src/themes/createColors.ts#L254), and it can easily become stale.